### PR TITLE
Use "audio track(s)" to refer to individual audio works 

### PR DESCRIPTION
**Pull request by Collins-Webdev**
**Update References to "Audio Track(s)" in English Text**

## Description
This pull request addresses issue #3972 by updating references to audio works to use the phrase "audio track(s)" instead of just "audio", following the convention established in issue #3941.

## Changes Made
- Replaced occurrences of "audio" with "audio track(s)" in the English text contained in en.json5.

- Ensured that the updated references maintain readability and consistency throughout the project.
- Verified the changes by running the project locally to ensure there are no visible errors.
- Reviewed and adhered to the established code style of the repository.
- No changes to tests or documentation were required for this update.

## Checklist

- [ ] My pull request has a descriptive title (not a vague title like`Update index.md`).
- [ ] My pull request targets the _default_ branch of the repository (`main`) or a parent feature branch.
- [ ] My commit messages follow [best practices][best_practices].
- [ ] My code follows the established code style of the repository.
- [ ] I added or updated tests for the changes I made (if applicable).
- [ ] I added or updated documentation (if applicable).
- [ ] I tried running the project locally and verified that there are no visible errors.
- [ ] I ran the DAG documentation generator (if applicable).

[best_practices]:
  https://git-scm.com/book/en/v2/Distributed-Git-Contributing-to-a-Project#_commit_guidelines

## Developer Certificate of Origin

<details>
<summary>Developer Certificate of Origin</summary>

```
Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
1 Letterman Drive
Suite D4700
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.


Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
```

</details>

### CODE REVIEW

**Comment by dhruvkb** [comment]
Hi @Collins-Webdev, could you update your PR description to use the [pull request template](https://github.com/WordPress/openverse/blob/main/.github/PULL_REQUEST_TEMPLATE/pull_request_template.md)? If you have any questions please let us know in the comments.

File: frontend/src/locales/scripts/en.json5 [hunk 1]
```diff
@@ -5,7 +5,7 @@
   },
   hero: {
     subtitle: "Explore more than 700 million creative works",
-    description: "An extensive library of free stock photos, images, and audio, available for free use.",
+    description: "An extensive library of free stock photos, images, and audio track, available for free use.",
```
Comment (dhruvkb) [inline frontend/src/locales/scripts/en.json5:8]: The word "track" was not needed here as we are referring to the entire body of audio works. Please revert.

File: frontend/src/locales/scripts/en.json5 [hunk 2]
```diff
@@ -80,14 +80,14 @@
     },
     collection: {
       content: {
-        a: "{openverse} searches across more than 700 million images and audio from open APIs and the {commonCrawl} dataset.",
+        a: "{openverse} searches across more than 700 million images and audio track from open APIs and the {commonCrawl} dataset.",
```
Comment (dhruvkb) [inline frontend/src/locales/scripts/en.json5:83]: We would use the plural form of the word here as we are referring to more than one audio track in this string.

```suggestion
        a: "{openverse} searches across more than 700 million images and audio tracks from open APIs and the {commonCrawl} dataset.",
```

File: frontend/src/locales/scripts/en.json5 [hunk 3]
```diff
@@ -80,14 +80,14 @@
...
         b: "We aggregate works from multiple public repositories, and facilitate reuse through features like one-click attribution.",
       },
     },
     planning: {
       content: {
         /** about.planning.content.a-c are parts of a single statement */
-        a: "Currently {openverse} only searches images and audio, with search for video provided through External Sources.",
+        a: "Currently {openverse} only searches images and audio track, with search for video provided through External Sources.",
```
Comment (dhruvkb) [inline frontend/src/locales/scripts/en.json5:90]: Please use the plural form.

```suggestion
        a: "Currently {openverse} only searches images and audio tracks, with search for video provided through External Sources.",
```

File: frontend/src/locales/scripts/en.json5 [hunk 4]
```diff
@@ -175,7 +175,7 @@
     },
     heading: {
       image: "Image Sources",
-      audio: "Audio Sources",
+      audio: "Audio track Sources",
```
Comment (dhruvkb) [inline frontend/src/locales/scripts/en.json5:178]: Please revert this change as "track" is not needed here. A source of audio tracks can be called an "audio source".

File: frontend/src/locales/scripts/en.json5 [hunk 5]
```diff
@@ -621,9 +621,9 @@
         countMore: "See over {localeCount} images found for {query}.",
       },
       audio: {
-        zero: "No audio found for {query}.",
-        count: "See {localeCount} audio found for {query}.",
-        countMore: "See over {localeCount} audio found for {query}.",
+        zero: "No audio track found for {query}.",
+        count: "See {localeCount} audio track found for {query}.",
+        countMore: "See over {localeCount} audio track found for {query}.",
```
Comment (dhruvkb) [inline frontend/src/locales/scripts/en.json5:626]: Please use the plural form.
```suggestion
        count: "See {localeCount} audio tracks found for {query}.",
        countMore: "See over {localeCount} audio tracks found for {query}.",
```

File: frontend/src/locales/scripts/en.json5 [hunk 6]
```diff
@@ -646,7 +646,7 @@
        * Interpolated into browsePage.searchForm.placeholder:
        * _{type}_ part of "Search all {type}"
        */
-      audio: "audio",
+      audio: "audio track",
```
Comment (dhruvkb) [inline frontend/src/locales/scripts/en.json5:649]: Please use the plural form.
```suggestion
      audio: "audio tracks",
```

File: frontend/src/locales/scripts/en.json5 [hunk 7]
```diff
@@ -871,14 +871,14 @@
   },
   audioDetails: {
     genreLabel: "Genre",
-    relatedAudios: "Related audio",
+    relatedAudios: "Related audio track",
```
Comment (dhruvkb) [inline frontend/src/locales/scripts/en.json5:874]: Please use the plural form.
```suggestion
    relatedAudios: "Related audio tracks",
```

File: frontend/src/locales/scripts/en.json5 [hunk 8]
```diff
@@ -930,19 +930,19 @@
...
+    ariaLabelInteractiveSeekable: "Audio: {title} - interactive player - press the space bar to play and pause a preview of the audio track; use the left and right arrow keys to seek through the track.",
     messages: {
       err_aborted: "You aborted playback.",
       err_network: "A network error occurred.",
-      err_decode: "Could not decode audio.",
+      err_decode: "Could not decode audio track.",
       err_unallowed: "Reproduction not allowed.",
       err_unknown: "An unexpected error has occurred. Try again in a few minutes or report the item if the issue persists.",
-      err_unsupported: "This audio format is not supported by your browser.",
+      err_unsupported: "This audio track format is not supported by your browser.",
```
Comment (dhruvkb) [inline frontend/src/locales/scripts/en.json5:941]: Please revert this change. The format used by the audio track is referred to as an "audio format".

File: frontend/src/locales/scripts/en.json5 [hunk 9]
```diff
@@ -930,19 +930,19 @@
...
       loading: "Loading...",
     },
     creator: "by {creator}",
-    close: "Close the audio player",
+    close: "Close the audio track player",
```
Comment (dhruvkb) [inline frontend/src/locales/scripts/en.json5:945]: Please revert this change as "audio player" is the right word to refer to the player which plays an audio track.

File: frontend/src/locales/scripts/en.json5 [hunk 10]
```diff
@@ -1153,9 +1153,9 @@
       },
       source: {
         audio: {
-          zero: "No audio files provided by this source",
-          count: "{count} audio file provided by this source|{count} audio files provided by this source",
-          countMore: "Over {count} audio files provided by this source",
+          zero: "No audio track files provided by this source",
+          count: "{count} audio track file provided by this source|{count} audio track files provided by this source",
+          countMore: "Over {count} audio track files provided by this source",
```
Comment (dhruvkb) [inline frontend/src/locales/scripts/en.json5:1158]: The word "track" and "file" should not both be used as one removes the need for the other.

```suggestion
          zero: "No audio tracks provided by this source",
          count: "{count} audio track provided by this source|{count} audio tracks provided by this source",
          countMore: "Over {count} audio tracks provided by this source",
```

File: frontend/src/locales/scripts/en.json5 [hunk 11]
```diff
@@ -1165,9 +1165,9 @@
       },
       tag: {
         audio: {
-          zero: "No audio files with the selected tag",
-          count: "{count} audio file with the selected tag|{count} audio files with the selected tag",
-          countMore: "Over {count} audio files with the selected tag",
+          zero: "No audio track files with the selected tag",
+          count: "{count} audio track file with the selected tag|{count} audio track files with the selected tag",
+          countMore: "Over {count} audio track files with the selected tag",
```
Comment (dhruvkb) [inline frontend/src/locales/scripts/en.json5:1170]: Same note as above^. The use of "track(s)" will eliminate the need for the word "file(s)".

File: frontend/src/locales/scripts/en.json5 [hunk 12]
```diff
@@ -1141,9 +1141,9 @@
     resultCountLabel: {
       creator: {
         audio: {
-          zero: "No audio files by this creator.",
-          count: "{count} audio file by this creator.|{count} audio files by this creator.",
-          countMore: "Over {count} audio files by this creator.",
+          zero: "No audio track files by this creator.",
+          count: "{count} audio track file by this creator.|{count} audio track files by this creator.",
+          countMore: "Over {count} audio track files by this creator.",
```
Comment (dhruvkb) [inline frontend/src/locales/scripts/en.json5:1146]: Same note as above^. The use of "track(s)" will eliminate the need for the word "file(s)".

File: frontend/src/locales/scripts/en.json5 [hunk 13]
```diff
@@ -991,7 +991,7 @@
   },
   searchType: {
     image: "Images",
-    audio: "Audio",
+    audio: "Audio track",
```
Comment (dhruvkb) [inline frontend/src/locales/scripts/en.json5:994]: Please revert this change as we are referring to to the collection of all audio tracks so "Audio" is suitable here.

File: frontend/src/locales/scripts/en.json5 [hunk 14]
```diff
@@ -1000,7 +1000,7 @@
     additional: "Coming soon",
     statusBeta: "Beta",
     seeImage: "See all images",
-    seeAudio: "See all audio",
+    seeAudio: "See all audio track",
```
Comment (dhruvkb) [inline frontend/src/locales/scripts/en.json5:1003]: Please use the plural form.
```suggestion
    seeAudio: "See all audio tracks",
```

**Review by dhruvkb — CHANGES_REQUESTED** [review]
The changes are good and you've made them in the exact right placc @Collins-Webdev! Thank you for the contributions.

However there are some grammatical issues that I have pointed out in the review. If you could please correct them that would be great. 

Let us know in the comments if you need any help. I will [convert this PR to a draft](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/about-pull-requests#draft-pull-requests) so that you can make the required changes and once you are done, you can [mark is as ready for review](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/changing-the-stage-of-a-pull-request#marking-a-pull-request-as-ready-for-review) again, thanks!

File: frontend/src/locales/scripts/en.json5 [hunk 15]
```diff
@@ -507,7 +507,7 @@
       title: "Source",
     },
     audioCategories: {
-      title: "Audio category",
+      title: "Audio track category",
```
Comment (obulat) [inline frontend/src/locales/scripts/en.json5:510]:
```suggestion
      title: "Audio category",
```

File: frontend/src/locales/scripts/en.json5 [hunk 16]
```diff
@@ -621,9 +621,9 @@
         countMore: "See over {localeCount} images found for {query}.",
       },
       audio: {
-        zero: "No audio found for {query}.",
-        count: "See {localeCount} audio found for {query}.",
-        countMore: "See over {localeCount} audio found for {query}.",
+        zero: "No audio track found for {query}.",
```
Comment (obulat) [inline frontend/src/locales/scripts/en.json5:624]:
```suggestion
        zero: "No audio tracks found for {query}.",
```

File: frontend/src/locales/scripts/en.json5 [hunk 17]
```diff
@@ -888,7 +888,7 @@
        * _{spacebar}_ part of "Press {spacebar} to play or pause the track."
        */
       spacebar: "Spacebar",
-    },
+    },
```
Comment (obulat) [inline frontend/src/locales/scripts/en.json5:891]:
```suggestion
    },
```

File: frontend/src/locales/scripts/en.json5 [hunk 18]
```diff
@@ -1142,7 +1142,7 @@
       creator: {
         audio: {
           zero: "No audio files by this creator.",
-          count: "{count} audio file by this creator.|{count} audio files by this creator.",
+          count: "{count} audio track file by this creator.|{count} audio track files by this creator.",
```
Comment (obulat) [inline frontend/src/locales/scripts/en.json5:1145]:
```suggestion
          count: "{count} audio track by this creator.|{count} audio tracks by this creator.",
```

File: frontend/src/locales/scripts/en.json5 [hunk 19]
```diff
@@ -1142,7 +1142,7 @@
       creator: {
         audio: {
           zero: "No audio files by this creator.",
```
Comment (obulat) [inline frontend/src/locales/scripts/en.json5:1144]:
```suggestion
          zero: "No audio tracks by this creator.",
```

File: frontend/src/locales/scripts/en.json5 [hunk 20]
```diff
@@ -1154,7 +1154,7 @@
       source: {
         audio: {
           zero: "No audio files provided by this source",
-          count: "{count} audio file provided by this source|{count} audio files provided by this source",
+          count: "{count} audio file provided by this source|{count} audio track files provided by this source",
```
Comment (obulat) [inline frontend/src/locales/scripts/en.json5:1157]:
```suggestion
          count: "{count} audio track provided by this source|{count} audio tracks provided by this source",
```

File: frontend/src/locales/scripts/en.json5 [hunk 21]
```diff
@@ -1166,7 +1166,7 @@
       tag: {
         audio: {
           zero: "No audio files with the selected tag",
-          count: "{count} audio file with the selected tag|{count} audio files with the selected tag",
+          count: "{count} audio file with the selected tag|{count} audio track files with the selected tag",
```
Comment (obulat) [inline frontend/src/locales/scripts/en.json5:1169]:
```suggestion
          count: "{count} audio track with the selected tag|{count} audio tracks with the selected tag",
```

**Comment by Collins-Webdev** [comment]
> Thank you for all your work on this, @Collins-Webdev ! I've added several more suggestions for changes. After adding them, could you mark this PR "Ready for review" so that we can approve and merge it?

Hello @obulat , 

It's a pleasure for me to be able to participate in such a large-scale project. I have taken your suggestions into account and made the necessary changes. Could you please review them and proceed with merging?
Thanks.

File: frontend/src/locales/scripts/en.json5 [hunk 22]
```diff
@@ -928,14 +928,14 @@
   audioThumbnail: {
     alt: 'Cover art for "{title}" by {creator}',
   },
-  audioTrack: {
+  audioTrack: {
```
Comment (obulat) [inline frontend/src/locales/scripts/en.json5:931]:
```suggestion
  audioTrack: {
```

File: frontend/src/locales/scripts/en.json5 [hunk 23]
```diff
@@ -1141,8 +1141,8 @@
     resultCountLabel: {
       creator: {
         audio: {
-          zero: "No audio files by this creator.",
-          count: "{count} audio file by this creator.|{count} audio files by this creator.",
+          zero: "No audio tracks by this creator.",
+          count: "{count} audio track by this creator.|{count} audio tracks files by this creator.",
```
Comment (obulat) [inline frontend/src/locales/scripts/en.json5:1145]:
```suggestion
          count: "{count} audio track by this creator.|{count} audio tracks by this creator.",
```

File: frontend/src/locales/scripts/en.json5 [hunk 24]
```diff
@@ -1154,7 +1154,7 @@
       source: {
         audio: {
           zero: "No audio files provided by this source",
-          count: "{count} audio file provided by this source|{count} audio files provided by this source",
+          count: "{count} audio track provided by this source|{count} audio tracks files provided by this source",
```
Comment (obulat) [inline frontend/src/locales/scripts/en.json5:1157]:
```suggestion
          count: "{count} audio track provided by this source|{count} audio tracks provided by this source",
```

File: frontend/src/locales/scripts/en.json5 [hunk 25]
```diff
@@ -1166,7 +1166,7 @@
       tag: {
         audio: {
           zero: "No audio files with the selected tag",
-          count: "{count} audio file with the selected tag|{count} audio files with the selected tag",
+          count: "{count} audio track with the selected tag|{count} audio tracks files with the selected tag",
```
Comment (obulat) [inline frontend/src/locales/scripts/en.json5:1169]:
```suggestion
          count: "{count} audio track with the selected tag|{count} audio tracks with the selected tag",
```

File: frontend/src/locales/scripts/en.json5 [hunk 26]
```diff
@@ -1177,4 +1177,4 @@
       },
     },
   },
-}
+}
```
Comment (obulat) [inline frontend/src/locales/scripts/en.json5:1180]:
```suggestion
}

```

File: frontend/src/locales/scripts/en.json5 [hunk 27]
```diff
@@ -1141,20 +1141,20 @@
...
+          count: "{count} audio track by this creator.|{count} audio tracks by this creator.",
           countMore: "Over {count} audio files by this creator.",
         },
         image: {
           zero: "No images by this creator.",
           count: "{count} image by this creator.|{count} images by this creator.",
           countMore: "Over {count} images by this creator.",
         },
-      },
+     },
```
Comment (obulat) [inline frontend/src/locales/scripts/en.json5:1153]:
```suggestion
      },
```

Comment (obulat) [reply frontend/src/locales/scripts/en.json5:1153]: Add missing space

**Review by obulat — CHANGES_REQUESTED** [review]
@Collins-Webdev, I added the whitespace changes to fix the linting step of the CI.

Now, the text of the `about` pages have changed, so we need to update the Visual regression testing snapshots. Here are instructions on how to do it: https://docs.openverse.org/frontend/guides/test.html#updating-snapshots

If you have any problems with this, please comment here - we can help or update for you.

**Review by obulat — APPROVED** [review]
I updated the failing snapshots, and the CI now passes.

Thank you for your contribution, @Collins-Webdev 🎉

**Review by dhruvkb — APPROVED** [review]
Thank you for the contribution @Collins-Webdev! Also thank you @obulat for making the pending changes.

**Pull request merged**